src/era/main.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python
+# * coding: utf8 *
+"""
+Updates the DWS ERAP layer based on their weekly
+"""
+
+import logging
+import sys
+from datetime import datetime
+from logging.handlers import RotatingFileHandler
+
+import arcgis
+from palletjack import ColorRampReclassifier, FeatureServiceInLineUpdater, SFTPLoader
+from supervisor.message_handlers import SendGridHandler
+from supervisor.models import MessageDetails, Supervisor
+
+from . import rotating, secrets
```

Comment:
I tried to leave everything I thought I could safely leave in `secrets_template.py`. Were you thinking of the service URLs/fieldnames/itemids? My thought was to leave them out because they could change, but I could copy them over to secrets_template.py if you think it's ok to have those in version control.

I guess my main thinking is that secrets.py kind of became a general settings file instead of breaking it out into a separate secrets-only file and a more general settings file. I prefer the simplicity of a single file, but I'm open to reasons why that's not the best idea.

Reply:
My suggestion is to separate secrets from configuration.